files/manifests/k8s-api-server.yaml

```diff
@@ -27,6 +27,9 @@ spec:
     {{ range .Hyperkube.Apiserver.Pod.CommandExtraArgs -}}
     - {{ . }}
     {{ end -}}
+    {{- if .MultiMasters.Enabled }}
+    - --apiserver-count=3
```

Comment:
For my understanding, why do we need to tell the apiserver how many replicas there are?

Reply:
I dont know, i would assume it needs it in order to not fuck up things? Its in the config so it might be good idea to set it up :p

```

--apiserver-count int     Default: 1
--
  | The  number of apiservers running in the cluster, must be a positive number.  

```

Reply:
the description is not that much usefull

Reply:
lol, w/e. thanks for pointers <3